builder/Makefile.tracee-container

```diff
@@ -129,14 +129,21 @@ build-alpine-tracee-nocore: \
 # run tracee (tracee-ebpf + tracee-rules)
 #
 
+DOCKER_RUN_ARGS = run --rm --pid=host --privileged \
+		-v /etc/os-release:/etc/os-release-host:ro \
+		-v $(PWD):/tracee \
+		-v /lib/modules:/lib/modules:ro \
+		-v /usr/src:/usr/src:ro \
+		-v /sys/kernel/security:/sys/kernel/security:ro \
```

Comment:
is this path present on every relevant distro? What happens if it isn't?

Reply:
Distros have to have CONFIG_SECURITYFS kconfig option defined so they can have the "securityfs" filesystem mounted in that directory.

```
$ mount -t securityfs securityfs /sys/kernel/security
```

From all distros I checked, and I'm currently testing tracee with, **ALL** of them have that directory (because all of them have CONFIG_SECURITYFS enabled) and have the "securityfs" filesystem mounted on it.

Problem with Alpine Linux is that it doesn't have CONFIG_SECURITY_LOCKDOWN_LSM=y on its kconfig options, but it does have CONFIG_SECURITYFS set (so the dir exists).

PS: Please let me know if you have a distro that doesn't have this set (I would be interested to understand why).

Reply:
That's fair enough. I was just curious to know. Since we know all of our supported distros have this, it's good enough 👍